codalab/apps/web/models.py

```diff
@@ -203,7 +202,8 @@ def _uuidify(directory):
     """Helper to generate UUID's in file names while maintaining their extension"""
     def wrapped_uuidify(obj, filename):
         name, extension = os.path.splitext(filename)
-        return os.path.join(directory, '{}-{}{}'.format(name, uuid.uuid4(), extension))
+        # Trims UUID down to 6 chars,
```

Comment:
fix this comment?

Maybe instead of a comment you can do this as well:

```
truncated_uuid = str(uuid.uuid4())[0:5]
truncated_name = name[0:35]
```